crates/egui/src/containers/scroll_area.rs

```diff
@@ -251,6 +254,19 @@ impl ScrollArea {
         self
     }
 
+    /// Control the dragging behavior
+    /// If `true` (default), the scroll area will respond to mouse dragging.
+    /// If `false`, the scroll area will not respond to mouse dragging
+    ///
+    /// This can be used, for example for code editors where
+    /// selecting and scrolling with a mouse is required.
+    ///
+    /// This controls both scrolling directions.
+    pub fn enable_mouse_dragging(mut self, enable: bool) -> Self {
```

Comment:
I think we should communicate that this is in particular for "drag to scroll". Perhaps it should be called `drag_to_scroll`?